plugins/gnupgplugin/gpgme_p.h

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (C) 2018 Damien Caliste.
+ * Contact: Damien Caliste <dcaliste@free.fr>
+ * All rights reserved.
+ * BSD 3-Clause License, see LICENSE.
+ */
```

Comment:
best to define a header guard even for private headers with known inclusion locations.